### PR TITLE
Fix cs

### PR DESCRIPTION
**Pull request by ScullWM**

### CODE REVIEW

**Review by EHER — APPROVED** [review]

**Comment by EHER** [comment]
Thanks @ScullWM ! ❤️

Would be nice to have something on the project to check the code style.

**Review by malukenho — CHANGES_REQUESTED** [review]
`php-cs` is too loose on checking. We should use `doctrine/coding-standard` because it has several advantages, as for example, it denies undefined `array` type usage.

**Comment by szepeviktor** [comment]
> `php-cs` is too loose on checking.

:) `doctrine/coding-standard` is a standard for PHPCS

`vendor/bin/phpcs --standard=Doctrine src/`

**Comment by szepeviktor** [comment]
I see! After looking an the code: php-cs-fixer is really too loose.

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
```
Comment (szepeviktor) [inline .github/workflows/php.yml:18]:
```suggestion
      run: composer validate --strict
```

**Review by malukenho — APPROVED** [review]

**Comment by malukenho** [comment]
I've made some changes to this PR:

- Use `doctrine/coding-standard`
- Rebase and Squased all commits
- Sign for participants
- Add validation `--strict` on composer as per @szepeviktor review

Thank you all! @ScullWM @szepeviktor 🎊

**Comment by malukenho** [comment]
@all-contributors please add @ScullWM  for code

**Pull request merged**